src/components/dropdown/_dropdown.scss

```diff
@@ -111,4 +111,16 @@
       }
     }
   }
+
+  .bx--dropdown--disabled {
+    background-color: rgba($field-01, 0.05);
```

Comment:
Hey! Thanks for adding this! However, could we simplify this and have it mimic our `text-input` disabled state?

You should just need to change this to
```scss
.bx--dropdown--disabled {
    opacity: .5;
    cursor: not-allowed;
}
```

And then you can remove the rest. Thanks!